app/rpc/namespaces/eth/api.go

```diff
@@ -65,6 +67,19 @@ func NewAPI(
 		panic(err)
 	}
 
+	//gasPrice: to get "minimum-gas-prices" config or to get ethermint.DefaultGasPrice
+	gasPricesStr := viper.GetString(server.FlagMinGasPrices)
```

Comment:
is `viper.GetString` always success? what's the default return value

Reply:
Empty string